packages/block-editor/src/components/global-styles/background-panel.js

```diff
@@ -372,7 +376,9 @@ function BackgroundImageControls( {
 
 	const onRemove = () =>
 		onChange(
-			setImmutably( style, [ 'background', 'backgroundImage' ], 'none' )
+			setImmutably( style, [ 'background' ], {
+				backgroundImage: 'none',
+			} )
```

Comment:
Here all background styles are wiped except "none".

The reason is that it should behave as though there is no image, and replacing the image shouldn't inherit any previously-set styles.